app/modules/wallet.js

```diff
@@ -99,34 +123,49 @@ export const retrieveTokensBalance = () => async (dispatch: DispatchType, getSta
   for (let [symbol] of TOKEN_PAIRS) {
     const scriptHash = getScriptHashForNetwork(net, symbol)
     // override scripthash with test if on test net
-    const [_error, tokenRpcEndpoint] = await asyncWrap(api.neonDB.getRPCEndpoint(net)) // eslint-disable-line
-    const [err, tokenResults] = await asyncWrap(api.nep5.getToken(tokenRpcEndpoint, scriptHash, address))
+    // eslint-disable-next-line
+    const [_error, tokenRpcEndpoint] = await asyncWrap(
+      api.neonDB.getRPCEndpoint(net)
+    )
+    const [err, tokenResults] = await asyncWrap(
+      api.nep5.getToken(tokenRpcEndpoint, scriptHash, address)
+    )
 
     if (!err) {
       tokens[symbol] = {
         ...tokenResults,
-        balance: tokenResults.balance === null ? 0 : tokenResults.balance,
+        balance:
+          tokenResults.balance === null
+            ? '0'
+            : toBigNumber(tokenResults.balance)
+              .toFixed(COIN_DECIMAL_LENGTH)
+              .toString(),
```

Comment:
On second thought, since these API responses are floats right now, I think we need to use `round` here instead of `toFixed`.

```js
tokenResults.balance === null
  ? '0'
  : toBigNumber(tokenResults.balance).round(COIN_DECIMAL_LENGTH).toString()
```

Reply:
would the same be true for GAS then, right?